xbmc/video/guilib/VideoActionProcessorHelper.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "VideoActionProcessorHelper.h"
+
+#include "FileItem.h"
+#include "GUIUserMessages.h"
+#include "ServiceBroker.h"
+#include "guilib/GUIComponent.h"
+#include "guilib/GUIWindowManager.h"
+#include "utils/log.h"
+#include "video/VideoDatabase.h"
+#include "video/dialogs/GUIDialogVideoVersion.h"
+
+using namespace VIDEO::GUILIB;
+
+CVideoActionProcessorHelper::~CVideoActionProcessorHelper()
```

Comment:
Is this 100% video version related? If so does it justify a rename?
Just asking since `CVideoActionProcessorHelper` is a really generic name which will likely serve to dump anything else in the future :)

Reply:
100% video

Reply:
In a future PR I will move resume selection dialog code which is currently duplicated to this helper, so 100% video but not 100% video version.